dcos/package.py

```diff
@@ -1340,13 +1340,10 @@ def _data(self, path):
         full_path = os.path.join(self.path, path)
         return util.read_file(full_path)
 
-    def package_versions(self):
-        """Returns all of the available package versions, most recent first.
+    def package_revisions(self):
```

Comment:
This still doesn't return the most recent first, since it's sorting strings, not ints